docs/manifest.json

```diff
@@ -1120,6 +1122,12 @@
       "path": "./guides/index.md",
       "icon_path": "./images/icons/notes.svg",
       "children": [
+        {
+          "title": "Using Organizations",
```

Comment:
This is the title that will appear at the top of the guide when it's rendered in the docs. If you want to include the (Alpha) or (Experimental) tag, it should also be present here.